app/config/bitcoin-devnet.json

```diff
@@ -0,0 +1,6 @@
+{
+	"bitcoinBroadcastLink": "",
```

Comment:
Please add a task and assign it to @sczembor : this should be our devnet endpoint

Reply:
https://github.com/gonative-cc/byield/issues/325